clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -47,7 +47,13 @@
      * <code>session.timeout.ms</code>
      */
     public static final String SESSION_TIMEOUT_MS_CONFIG = "session.timeout.ms";
-    private static final String SESSION_TIMEOUT_MS_DOC = "The timeout used to detect failures when using Kafka's group management facilities.";
+    private static final String SESSION_TIMEOUT_MS_DOC = "The timeout used to detect failures when using Kafka's consumer " +
+            "group management facilities. When the session timeout expires on the broker without having received a heartbeat " +
```

Comment:
If we remove `consumer` from the group management part, does it make sense to mention it in the heartbeat part?